src/components/shared/MarkerListView/index.js

```diff
@@ -333,10 +348,11 @@ class MarkerListView extends Component<Props, State> {
 
   render() {
     const { items, supportedNavigationModes, userLocation } = this.props;
-    const { selectedNavigationMode, activeMarker } = this.state;
+    const { selectedNavigationMode, activeMarker, shouldShowInstructions } = this.state;
 
     return (
       <View style={styles.container}>
+        {shouldShowInstructions && <UsageModal onRequestClose={() => { this.closeInstructions(); }} />}
```

Comment:
`onRequestClose={() => { this.closeInstructions(); }}` could be written as `onRequestClose={() =>  this.closeInstructions()}`

Reply:
I can't remember what ESLint had to say about that. I'll check.